### DISCUSSION TITLE
Navigasjonskort hover

### DISCUSSION THREAD

**Discussion by ivnai** [fremtind/jokul#3398] [Annet]
Hei. Er det flere som synes 2-retning mosjon på kort virker komplisert? https://jokul.fremtind.no/komponenter/card
Hover: pilen drar øst mens kortet nord. Foreslår å fjerne bevegelse til pil. Eller?
@Murstam @Stakset @Slaine2 @MonaHolm72 osv.

**Comment by Murstam** [comment]
Ja, helt enig, to bevegelser i ulik retning er rotete. Denne har vi diskutert før, slik jeg husker det ble vi enige om å fjerne bevegelse i pil og beholde på kort?

**Reply by Slaine2** [reply]
Ok, la oss da fjerne bevegelsen i kartet i Jøkul.

**Reply by wkillerud** [reply]
Laget et issue:
- https://github.com/fremtind/jokul/issues/3400
